apps/docs/app/ui/layout/header/Header.tsx

```diff
@@ -1,20 +1,100 @@
 "use client";
```

Comment:
![image](https://github.com/user-attachments/assets/6d0128e6-9d0c-453a-a659-38573912b5d9)
Should IDP have a section title as well?